smart-contracts/osmosis/contracts/cl-vault/tests/test-tube/autocompound.rs

```diff
@@ -287,8 +288,8 @@ fn test_autocompound_with_rewards_swap_non_vault_funds() {
         &ExecuteMsg::VaultExtension(ExtensionExecuteMsg::SwapNonVaultFunds {
             swap_operations: vec![SwapOperation {
                 token_in_denom: DENOM_REWARD.to_string(),
-                pool_id_0: swap_pools_ids[2],
-                pool_id_1: swap_pools_ids[1],
+                pool_id_token_0: swap_pools_ids[2],
```

Comment:
I don't think that adding token to the name does provide additional information. So would prefer to either keep the old naming or replace the _0, _1-prefixes. If that is referring to pool tokens, then I'd prefer to use base and quote as pre- or suffix as these are most widely used (though I always confuse them so we would need to document that:) and have a clear meaning